### DISCUSSION TITLE
How to use this package with VueJS 3 SPA ?

### DISCUSSION THREAD

**Discussion by lalithhakari** [lmsqueezy/lemonsqueezy.js#33] [Q&A]
My backend is powered by Laravel and using latest [LemonSqueezy](https://github.com/lmsqueezy/laravel) .
Frontend is powered by VueJS v3 with TS.
Steps i have done till now
1. `npm install @lemonsqueezy/lemonsqueezy.js`
2. `<script setup lang="ts">
     import LemonSqueezy from "@lemonsqueezy/lemonsqueezy.js"
     </script>`
By just following these 2 steps in frontend, im facing this error
"**[plugin:vite:import-analysis] Failed to resolve entry for package "@lemonsqueezy/lemonsqueezy.js". The package may have incorrect main/module/exports specified in its package.json.**"
<img width="914" alt="Screenshot 2023-12-10 at 12 21 06 AM" src="https://github.com/lmsqueezy/lemonsqueezy.js/assets/53531525/e5978dab-8de3-4802-ac8a-92b2340052ba">

**Comment by lalithhakari** [comment]
I suspect that this issue is linked to [Already Reported Issue](https://github.com/lmsqueezy/lemonsqueezy.js/issues/32)

This is the first time Im trying out LemonSqueezy. Am i missing something?

**Comment by lalithhakari** [comment]
I have rechecked the docs. And Im not missing anything! [https://github.com/lmsqueezy/lemonsqueezy.js#installation](https://github.com/lmsqueezy/lemonsqueezy.js#installation)

**Comment by AF1QUE** [comment]
@lalithhakari were you able to make it work? I am using react and facing the same issue.

**Comment by brankoconjic** [comment]
This has been fixed with the 1.2.5 version.
